lib/kafka-consumer.js

```diff
@@ -101,65 +99,60 @@ Consumer.prototype._onWrapper = function (consumerInstance, eventName, cb) {
     return consumerInstance.__kafkaAvro_on(eventName, cb);
   }
 
-  return consumerInstance.__kafkaAvro_on('data', function(message) {
-    if (!this.sr.keySchemas[message.topic]) {
```

Comment:
That's because I've moved away from this mechanism in favor of just extracting schemaId from the magic byte. There is the same warning being logged a couple of lines below though.